CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
     IconThemes
     XmlGui
     WindowSystem
+    KDELibs4Support
```

Comment:
Why is this needed? The goal is to have no KDE4 dependencies. Please mention the system where the build fails and the error message in the commit message. (This helps communicate *why* this change is good.)

Reply:
Oh, I am sorry. I was glad to see kdbg ported to KF5 and hurried too much )
System is unstable gentoo with kdeframeworks 5.37.0. First error was
/tmp/build/kdbg/kdbg/pgmargs.cpp:8:34: fatal error: klocale.h: No such file or directory
The file klocale.h can be found in kdelibs4support. But you are right, this is not good fix. Using klocalizedstring.h instead of klocale.h will be better.